account_invoice_refund_link/models/account_invoice_line.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    origin_line_ids = fields.Many2many(
+        comodel_name='account.invoice.line', column1='refund_line_id',
+        column2='original_line_id', string="Original invoice line",
+        relation='account_invoice_line_refunds_rel',
+        help="Links to original invoice line which is referred by current "
```

Comment:
check previous messages (I would even simplify and make it like "Original invoice related to this refund")